relation_engine_server/api_versions/api_v1.py

```diff
@@ -71,14 +71,13 @@ def run_query():
      - public stored queries (these have access controls within them based on params)
     """
     json_body = parse_json.get_json_body() or {}
-    # Don't allow the user to set the special 'ws_ids' field
-    json_body['ws_ids'] = []
-    auth_token = auth.get_auth_header()
     # Fetch any authorized workspace IDs using a KBase auth token, if present
+    auth_token = auth.get_auth_header()
     ws_ids = auth.get_workspace_ids(auth_token)
     # fetch number of documents to return
     batch_size = int(flask.request.args.get('batch_size', 10000))
     full_count = flask.request.args.get('full_count', False)
+
```

Comment:
You don't have to revert, but i prefer not having all these additional newlines in the function body

Reply:
I find adding newlines aids readability -- otherwise it's just a bunch of lines with the same indentation and there's added cognitive load in understanding the code before you can edit it.